src/utils/array/shuffleArray/shuffleArray.ts

```
@@ -0,0 +1,8 @@
+export function shuffleArray<T>(arr: T[]): T[] {
+    const arrayCopy =[...arr];
+    for (let i=arrayCopy.length-1;i>0;i--){
+        const j =Math.floor(Math.random()*(i+1));
+        [arrayCopy[i],arrayCopy[j]]=[arrayCopy[i],arrayCopy[j]];
```

Biome

[error] i is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)